components/git/git-node-help

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env node
 
 console.log(`
+Steps to get metadata for a pull request:
+==============================================================================
+$ cd path/to/node/project
+$ git node metadata $PRID        # Get metadata for the specified PR
+==============================================================================
```

Comment:
Can you make this consistent with current help msg we have at here.
https://github.com/nodejs/node-core-utils/blob/f8a96584d78786e99ba63dfcccba3109f656556b/lib/args.js#L14-L17